test/e2e/categories_test.go

```diff
@@ -38,7 +38,7 @@ const (
 	defaultNonExistingAdditionalCategoryValue = "nonExistingCategoryValueCAPX"
 )
 
-var _ = Describe("Nutanix categories [PR-Blocking]", func() {
+var _ = Describe("Nutanix categories", func() {
```

Comment:
is this not considered pr blocking anymore?

Reply:
correct, in my opinion its not as this is just setting additional categories. also we need to change the name in the describe to correct reflect what the test does